src/containers/SuccessForm.tsx

```diff
@@ -1,18 +1,17 @@
-import { useContext } from "react";
 import { Link } from "react-router-dom";
+import { useCookies } from "react-cookie";
 
 import { PrimaryButton } from "../components/Buttons";
-import Context from "../store/Context";
 
 const SuccessForm = () => {
-  const [{ user }] = useContext(Context);
+  const [{ sessionInfo }] = useCookies();
```

Comment:
i'm just going to keep track of this as a cookie for now, since we aren't _really_using any session info at all beyond just knowing if they've already signed up